database/sqlite/sqlite_aclk_alert.c

```diff
@@ -43,6 +43,34 @@ void update_filtered(ALARM_ENTRY *ae, uint32_t unique_id, char *uuid_str) {
     ae->flags |= HEALTH_ENTRY_FLAG_ACLK_QUEUED;
 }
 
+static inline bool is_event_from_alert_variable_config(uint32_t unique_id, char *uuid_str) {
+    sqlite3_stmt *res = NULL;
+    int rc = 0;
+    bool ret = false;
+
+    char sql[ACLK_SYNC_QUERY_SIZE];
+    snprintfz(sql,ACLK_SYNC_QUERY_SIZE-1, "select hl.unique_id from health_log_%s hl, alert_hash ah where hl.unique_id = %u " \
+                                          "and hl.config_hash_id = ah.hash_id " \
```

Comment:
@stelfrag this query is showing that we are almost reaching the level where to use foreign key are more than necessary. :thinking: